etl/steps/data/garden/wb/2024-01-17/shared.py

```diff
@@ -0,0 +1,855 @@
+
+                # Replace placeholders
+                tb_garden[col_name].metadata.description_short = (
+                    tb_garden[col_name]
+                    .metadata.description_short.replace("{povline}", rel_dict[rel])
+                    .replace("{inc_cons_dict[wel]['name']}", inc_cons_dict[welfare_type]["name"])
+                )
+
+        for pct in pct_dict:
+            # For variables that use percentiles (deciles)
+            col_name = f"decile{pct}_{var}"
+
+            if col_name in cols:
+                # Get the origins of the variable
+                origins = tb_garden[col_name].metadata.origins
+
+                # Create metadata for these variables
+                tb_garden[col_name].metadata = var_metadata_percentiles(var, pct, origins, ppp_version, welfare_type)
+
+                # Replace placeholders
+                tb_garden[col_name].metadata.description_short = (
+                    tb_garden[col_name]
+                    .metadata.description_short.replace("{str(pct)}", f"{str(pct)}0")
+                    .replace(
+                        "{inc_cons_dict[wel]['name_distribution']}",
+                        inc_cons_dict[welfare_type]["name_distribution"],
+                    )
+                    .replace("{inc_cons_dict[wel]['verb']}", inc_cons_dict[welfare_type]["verb"])
+                    .replace("{pct_dict[pct]['decile10']}", pct_dict[pct]["decile10"].lower())
+                )
+
+                tb_garden[col_name].metadata.description_key = [
+                    ppp.replace("{ppp}", str(ppp_version)) for ppp in tb_garden[col_name].metadata.description_key
+                ]
+                tb_garden[col_name].metadata.unit = tb_garden[col_name].metadata.unit.replace("{ppp}", str(ppp_version))
+
+    return tb_garden
+
+
+# Metadata functions to show a clearer main code
+def var_metadata_inequality_mean_median(var, origins, welfare_type) -> VariableMeta:
+    """
+    Create metadata for defined uniquely by their name
+    """
+    # For monetary variables I include PPP description
+    if var in ["mean", "median"]:
+        meta = VariableMeta(
+            title=f"{var_dict[var]['title']} {inc_cons_dict[welfare_type]['name']}",
+            description_short=var_dict[var]["description"],
+            description_key=[
+                ppp_description,
+                inc_cons_dict[welfare_type]["description"],
+                non_market_income_description,
+            ],
+            description_processing=f"""{inc_cons_dict[welfare_type]['processing_description']}""",
+            unit=var_dict[var]["unit"],
+            short_unit=var_dict[var]["short_unit"],
+            origins=origins,
+        )
+        meta.display = {
+            "name": meta.title,
+            "numDecimalPlaces": var_dict[var]["numDecimalPlaces"],
+            "tolerance": TOLERANCE,
+        }
+
+        meta.presentation = {
```

Comment:
Nit - to fix the linting error, you should use proper object, i.e.
```
from owid.catalog import VariablePresentationMeta
...
meta.presentation = VariablePresentationMeta(title_public=meta.title)
```

Reply:
Thanks! By the way, I don't have that linting error

Reply:
Oh weird, could you check if you have `Pylance` extension enabled in VSCode? (It's not obligatory, but it's pretty useful)

Reply:
It's enabled. Should I check a specific place in settings as well?